portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionCustomElement.testcase

```diff
@@ -0,0 +1,65 @@
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Remote Apps";
+	property testray.main.component.name = "Remote Apps";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			RemoteApps.tearDown();
+
+			Navigator.gotoPage(pageName = "Home");
+		}
+	}
+
+	@description = "LPS-170678 - Assert if 'Custom Element' can be categorized as 'Client Extension'"
+	@priority = "3"
+	test CanBeCategorizedInAddWidgetList {
+		property app.server.types = "tomcat";
+		property database.types = "mysql";
+		property environment.acceptance = "true";
+		property portal.acceptance = "true";
+
+		task ("When client extension has portal category 'Client Extensions'") {
+			RemoteApps.goToRemoteAppsPortlet();
+
+			RemoteApps.addCustomElementAllFields(
+				entryCssurl = "https://liferay.github.io/liferay-frontend-projects/index.css",
+				entryHtmlName = "vanilla-counter",
+				entryName = "Vanilla Counter",
+				entryProperties = "test-data-user=QAuser",
+				entryURL = "https://liferay.github.io/liferay-frontend-projects/vanilla-counter/index.js");
+		}
+
+		task ("And When on Content Page") {
+			Navigator.openURL();
+
+			Click(locator1 = "//span[@title='Edit']");
```

Comment:
Hey @mateussandes23 Could you put some existing path here or create if it does not exist?

Reply:
Replace with ContentPagesNavigator.openEditContentPage

Reply:
Ok, I'm working on it.